.circleci/config.yml

```diff
@@ -13,6 +13,15 @@ references:
         at: ~/grakn
 
 jobs:
+  lint:
+    machine: true
+    working_directory: ~/grakn
+    steps:
+      - checkout
+      - *install_bazel
+      - run: python ./verify.py
+      - run: bazel test $(bazel query 'kind(checkstyle_test, //...)') --test_output=errors
```

Comment:
Can you explain to me what this code does?

Reply:
@lolski there're two parts:
1. `bazel query 'kind(checkstyle_test, //...)'` means "from *all* (`//...`) targets, get me these that are of kind `checkstyle_test`"
2. `bazel test` the result of (1), outputting the errors only